### DISCUSSION TITLE
Is this supported by a business?

### DISCUSSION THREAD

**Discussion by coltenkrauter** [cdklabs/cdk-nextjs#14] [General]
Is this an official package built by AWS? 

I see the org is **cdklabs** and the associated email is [aws-cdk-dev@amazon.com](mailto:aws-cdk-dev@amazon.com). I am curious to learn more about who is building this.

**Comment by bestickley** [comment]
Hey @coltenkrauter, I've built this construct with inspiration/experience from cdk-nextjs-standalone which previously I helped maintain. Also learned a lot from open-next's docs too and talking to their maintainers. I'm a part of AWS Professional Services. I've been dogfooding this construct with web apps I'm building for customers. So to answer your question: yes. I've got a team of other developers who will be helping me too (@rayjanwilson, @k-was). And of course, we're interested in help from the community :)

**Reply by coltenkrauter** [reply]
Very cool, thanks for the info. I am certainly interested in contributing.

Is the `cdklabs` GitHub org affiliated or owned by AWS or Amazon?

**Reply by bestickley** [reply]
Yes, cdklabs is owned by AWS Open Source Team.
